### DISCUSSION TITLE
Warning: "aten::std mean.correction" M1 Pro

### DISCUSSION THREAD

**Discussion by stepkono** [lllyasviel/Fooocus#1206] [Q&A]
Hey guys, does anyone know how to fix the following issue? The image gets rendered, however it brings up this warning, which obviously results in machine slipping back to CPU rendering, even though ive got PyTorch nightly installed which - I would assume - would use mps backend. 
I saw similar questions in other discussions, but unfortunately they've never got clearly answered. 

The warning: 
"The operator 'aten::std mean.correction' 
is not currently supported on the MPS backend and will fall back to run on the CPU. This may have performance implications."

<img width="1512" alt="Warning" src="https://github.com/lllyasviel/Fooocus/assets/152878962/ba7c5956-34b1-431c-80ae-bcdc9c1a104b">

**Comment by m00r3ik** [comment]
This [doesn't seem to be implemented](https://github.com/pytorch/pytorch/issues/77764) in torch yet.  
Unfortunately ...
